.github/workflows/test-failures-report.yml

```
@@ -1,35 +1,36 @@
-          persist-credentials: false
-      - name: Download reports for each test job
-        uses: dawidd6/action-download-artifact@v21
-        with:
-          run_id: ${{ github.event.workflow_run.id }}
-          name: test-reports
-          merge_multiple: true
-      - name: Test Report
+          path: artifacts
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id }}
+      - name: Test Report for ${{ needs.resolve.outputs.title }}
```

actionlint (1.7.12)

[error] property "resolve" is not defined in object type {}

(expression)